src/main/java/com/patchworkmc/manifest/converter/accesstransformer/AccessTransformerConverter.java

```diff
@@ -0,0 +1,132 @@
+ */
+public class AccessTransformerConverter {
+	private AccessTransformerConverter() {
+		// NO-OP
+	}
+
+	public static byte[] convertToWidener(ForgeAccessTransformer accessTransformer, IntermediaryHolder holder) {
+		StringBuilder sb = new StringBuilder();
+		sb.append("accessWidener\tv1\t").append("intermediary").append('\n');
+		sb.append("#\tAutomatically generated by Patchwork Patcher.\n");
+		sb.append("#\tThe original (but remapped) AT entry is roughly preserved here for debugging.\n");
+		sb.append("#\tThey may be stripped in future releases.\n\n");
+
+		for (TransformedClass targetClass : accessTransformer.getClasses()) {
+			if (targetClass.getAccessLevel() != AccessLevel.KEEP || targetClass.getFinalization() != Finalization.KEEP) {
+				writeClass(sb, targetClass);
+			}
+			writeWildcards(sb, targetClass.getName(), (TransformedWildcardMember) targetClass.getDefaultForFields(),
+				(TransformedWildcardMember) targetClass.getDefaultForMethods(), holder);
+
+			targetClass.getFields().forEach(field -> writeField(sb, field, holder));
+			targetClass.getMethods().forEach(method -> writeMethod(sb, method));
+		}
+
+		return sb.toString().getBytes();
+	}
+
+
+	private static void writeClass(StringBuilder sb, TransformedClass targetClass) {
+		//// Debug
+		// Modifer word
+		writeDebugModifier(sb, targetClass);
+		// Target
+		sb.append('\t').append(targetClass.getName()).append('\n');
+
+		//// Widener
+		// Modifier word
+		sb.append(targetClass.getFinalization() == Finalization.REMOVE ? "extendable" : "accessible");
+		// Target
+		sb.append("\tclass\t").append(targetClass.getName()).append('\n');
+	}
+
+	private static void writeWildcards(StringBuilder sb, String owner, TransformedWildcardMember fields, TransformedWildcardMember methods, IntermediaryHolder holder) {
+		//// Debug
+		if (fields != null) {
+			writeDebugModifier(sb, fields);
+			sb.append('\t').append(owner).append('\t').append("*\n");
+		}
+		if (methods != null) {
+			writeDebugModifier(sb, methods);
+			sb.append('\t').append(owner).append('\t').append("*\n");
+		}
+
+
+		//// Widener
+		for (Map.Entry<String, IntermediaryHolder.Member> entry : holder.getMappings().get(owner).entrySet()) {
+			IntermediaryHolder.Member member = entry.getValue();
+
+			if (member.isField && fields != null) {
+				// We don't write the debug because it's already been done
+				writeField(sb, owner, member.name, member.descriptor, fields, false);
+			} else if (methods != null) {
+				writeMethod(sb, owner, member.name, member.descriptor, methods, false);
+			}
+		}
+	}
+
+	private static void writeField(StringBuilder sb, TransformedField transformed, IntermediaryHolder holder) {
+		String owner = transformed.getOwner();
+		String name = transformed.getName();
+		String descriptor = holder.getMappings().get(owner).get(name).descriptor;
+		AccessTransformerConverter.writeField(sb, owner, name, descriptor, transformed, true);
```

Comment:
Are you sure that the qualification is needed?
```suggestion
		writeField(sb, owner, name, descriptor, transformed, true);
```